src/ast/passes/pid_filter_pass.cpp

```diff
@@ -65,16 +66,24 @@ static BlockExpr *create_pid_filter(ASTContext &ast,
                                     int pid,
                                     BlockExpr *orig_block)
 {
+  auto *ret_block = ast.make_node<BlockExpr>(
+      orig_block->loc,
+      StatementList(
+          { ast.make_node<Jump>(orig_block->loc,
+                                ast::JumpType::RETURN,
+                                ast.make_node<None>(orig_block->loc)) }),
```

Comment:
Do you want this? I think you just want `make_node<Jump>(orig_block->loc, ast::JumpType::RETURN)`? This will specify a return value of `None`. That's weird, I would expect this to be an error in codegen. It would be like `return void_function();`